src/unix/tcp.c

```diff
@@ -225,9 +227,33 @@ static int uv__is_ipv6_link_local(const struct sockaddr* addr) {
 
 static int uv__ipv6_link_local_scope_id(void) {
   struct sockaddr_in6* a6;
+  int rv;
+  // AIX path fallback to use uv_interface_addresses
+#if defined (_AIX) && !defined(__PASE__)
```

Comment:
No C99 comments and if you're going to effectively replace the function with another then for legibility reasons just do so. That is:
```c
static int uv__ipv6_link_local_scope_id(void) {
#if defined (_AIX) && !defined(__PASE__)
  // ...
#else
  // ...
#endif  /* defined (_AIX) && !defined(__PASE__) */
}
```